### PR TITLE
[BER-FT-032021] Pietro Ballarin

### PR DESCRIPTION
**Pull request by pietroballarin**
Hi. I would like to have some feedback about this first draft of the assignment .

- I am still unsure about the use of flex-box and, in particular, if the search bar section has been set correctly
- I am not able to make the headings as bold as in the pictures 
- I divided the button from the boxes in the last section by using two separate divs. I don't know if it's the right choice or if there is a different solution to separate them.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,9 +1,94 @@
...
-your team, and start collaborating. Nothing to configure, set up, or manage. Team management Control who has access to
-what modules within your team namespace using straightforward team management capabilities. Familiar features npm Orgs
-has 100% parity with all the public npm registry features your developers already use. npm audit Enjoy the security
-auditing features built into the npm client, a zero-friction way to make open source software safer. Create an Org
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title> NPM </title>
+      <link rel="stylesheet" href="style.css">
```
Comment (wassbeer) [inline index.html:6]: Please be aware of the indentation here. One tab less would do.

File: index.html [hunk 2]
```diff
@@ -1,9 +1,94 @@
...
+
```
Comment (wassbeer) [inline index.html:7]: Does this empty line exist for a reason? :)

File: index.html [hunk 3]
```diff
@@ -1,9 +1,94 @@
...
+     <div>
+      <a href="#" class="button3">Create an Org</a>
+     </div>
+    </section>
+    
+  </body>
+</html>
+      
+     
+
```
Comment (wassbeer) [inline index.html:93]: Do these empty lines exist for a reason? :)

File: style.css [hunk 4]
```diff
@@ -0,0 +1,253 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
```
Comment (wassbeer) [inline style.css:1]: Also in this document, please be mindful about the use of empty lines. They should only be used to indicate a division between two independent 'parts' (e.g. classes, IDs, HTML blocks, etc.).

File: style.css [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+#features .feature-list {
+  display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-evenly;
+	align-items: center;
+	align-content: center;
+}
+
+#features .button3 {
```
Comment (wassbeer) [inline style.css:240]: This button is bigger than in the example. No major problem, just a detail that could be improved. :)

File: style.css [hunk 6]
```diff
@@ -0,0 +1,253 @@
...
+nav ul {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  list-style: none;
+}
+
+nav a {
+  text-decoration: none;
+  color: black;
```
Comment (wassbeer) [inline style.css:33]: In your CSS, different types of colour declarations (human language, RGB and Hexadecimal) are used. Try to stick to one.

**Review by wassbeer — COMMENTED** [review]
Great stuff Pietro!
Please check out my comments for potential improvements.

Thomas (TA WDFT).

File: style.css (hunk 4 above)
Comment (pietroballarin) [reply style.css:1]: The empty lines exist for no particular reason. I'll be more careful to avoid leaving them.

File: style.css (hunk 6 above)
Comment (pietroballarin) [reply style.css:33]: Got it, thanks.

File: style.css (hunk 5 above)
Comment (pietroballarin) [reply style.css:240]: Sure, I'll adapt it :) 
I noticed though, that I get different results according to the size of the screen. Is there a way to resize the content according to the screen resolution, for example, if I watch it on my laptop or my phone?

**Comment by wassbeer** [comment]
Yes, Pietro. You can make your page responsive by making use of media
queries and other tools such as a library with a responsive grid system. :)
If you are already curious about that, you can Google "Responsive web
design with CSS" and dive into it. Cheers!

**Pull request closed without merging**